### ISSUE TITLE
about the psnr and ssim on ucf101 dataset

### ISSUE THREAD

**Issue by redrock303** [tding1/CDFI#7]
I tested the results provided in your rep, but I got 34.11/0.9411 on the ucf101 test set using the frame_01_ours.png as your network output and frame_01_gt.png as the reference. However, it doesn't match the result reported in your paper. Is there something wrong with my testing code? Thanks!
![image](https://user-images.githubusercontent.com/24226074/114514413-d8f16880-9c6d-11eb-9ede-b4cd6ba5428e.png

**Comment by tding1** [comment]
Please note that the "frame_01_ours.png" is not the output from our network. All the images under the folder "ucf101_interp_ours" are originated from [here](https://github.com/liuziwei7/voxel-flow). To reproduce our results, you need to first load "frame00.png" and "frame02.png", and use our pre-trained model to generate the actual output from our network, then compare it with the "frame_01_gt.png". Please take a look at our testing framework, see [here](https://github.com/tding1/CDFI/blob/23a0f6ed56d1e9c67fdcfe3fe2ca5f482609ca6a/test.py#L86-L98).
